newIDE/app/src/AssetStore/AssetCard.js

```diff
@@ -19,6 +19,11 @@ const styles = {
     verticalAlign: 'middle',
     pointerEvents: 'none',
   },
+  previewImagePixelated: {
+    width: '-webkit-fill-available',
+    imageRendering: 'pixelated',
```

Comment:
Let's prefer crisp-edges, as it works in most browsers and is recommended in https://developer.mozilla.org/en-US/docs/Web/CSS/image-rendering for pixel art.

Reply:
`crisp-edges` is blurry and not found in chrome dev tools.
![image](https://user-images.githubusercontent.com/1670670/127166431-6ba3a4be-f41c-4de6-a839-d1253b43f116.png)